engine/apps/metrics_exporter/metrics_collectors.py

```diff
@@ -26,42 +29,63 @@
 
 RE_ALERT_GROUPS_TOTAL = re.compile(r"{}_(\d+)".format(ALERT_GROUPS_TOTAL))
 RE_ALERT_GROUPS_RESPONSE_TIME = re.compile(r"{}_(\d+)".format(ALERT_GROUPS_RESPONSE_TIME))
+RE_USER_WAS_NOTIFIED_OF_ALERT_GROUPS = re.compile(r"{}_(\d+)".format(USER_WAS_NOTIFIED_OF_ALERT_GROUPS))
 
 
 # https://github.com/prometheus/client_python#custom-collectors
 class ApplicationMetricsCollector:
     def __init__(self):
         self._buckets = (60, 300, 600, 3600, "+Inf")
-        self._labels = [
-            "integration",
-            "team",
+        self._stack_labels = [
             "org_id",
             "slug",
             "id",
         ]
-        self._labels_with_state = self._labels + ["state"]
+        self._integration_labels = [
+            "integration",
+            "team",
+        ] + self._stack_labels
+        self._integration_labels_with_state = self._integration_labels + ["state"]
+        self._user_labels = ["username"] + self._stack_labels
 
     def collect(self):
-        alert_groups_total = GaugeMetricFamily(ALERT_GROUPS_TOTAL, "All alert groups", labels=self._labels_with_state)
-        alert_groups_response_time_seconds = HistogramMetricFamily(
-            ALERT_GROUPS_RESPONSE_TIME, "Users response time to alert groups in 7 days (seconds)", labels=self._labels
+        org_ids = set(get_organization_ids())
+
+        # alert groups total metric: gauge
+        alert_groups_total, missing_org_ids_1 = self.get_alert_groups_total_metric_and_missing_org_ids(org_ids)
```

Comment:
nit: I would consider renaming `get_alert_groups_total_metric_and_missing_org_ids` to `_get_alert_groups_total_metric`, to keep the name shorter/easier to read and indicate that this is an internal-only helper (same for the others)